strelka/strelka.cwl

```diff
@@ -9,7 +9,7 @@ requirements:
         coresMin: 8
```

Comment:
since lookup values and variables may not work in the resourceRequirements section we could hard-code this to 4.